manual/lib.typ

```diff
@@ -0,0 +1,439 @@
+
+// ================================
+// 字体配置
+// ================================
+
+#let font = (
+  // 中文字体
+  zh_shusong: "Noto Serif CJK SC",
+  zh_zhongsong: "Noto Serif CJK SC",
+  zh_kai: "LXGW WenKai",
+  zh_hei: "Noto Sans CJK SC",
+  // 英文字体
+  en_sans_serif: "New Computer Modern",
+  en_serif: "New Computer Modern",
+  en_typewriter: "Courier New",
+  en_code: "Consolas",
```

Comment:
Those two fonts are proprietary, non-free fonts. Since Typst will always embed fonts[^embed], using it may cause legal issues. If you really want this style monospace font, please use 'Courier Prime', which uses the same style, is free, and metric-compatible with 'Courier'.

[^embed]: https://github.com/typst/typst/issues/6466